changelog/unreleased/kong/optimize_keepalive_parameters.yml

```diff
@@ -0,0 +1,3 @@
+message: Bumped default values of `nginx_http_keepalive_requests` and `upstream_keepalive_max_requests` to `10000`.
+type: performance
+scope: Configuration
```

Comment:
Although this PR modifies the default value of NGINX, it doesn't have a significant impact on the client's side. Therefore, I don't believe we need to categorize this feature as a breaking_change.

Typically, users won't notice this alteration. However, under stressful testing scenarios, they might observe an increase in NGINX's performance, specifically in RPS